src/fsharp/Microsoft.DotNet.DependencyManager/NativeDllResolveHandler.fsi

```diff
@@ -12,6 +12,6 @@ type NativeResolutionProbe = delegate of Unit -> seq<string>
 type NativeDllResolveHandler =
 
     /// Construct a new NativeDllResolveHandler
-    new: nativeProbingRoots: NativeResolutionProbe -> NativeDllResolveHandler
```

Comment:
I think the underscore might have just been a mistake, since the value is actually used. The source could probably just be updated to not have the underscores, no?

Reply:
The underscore here is, in fact, functional in nature - due to the fact that the implementation's use of the argument is conditional, removing the underscore triggers a not-used warning in some cases.

Reply:
oh boy, what a dirty implementation. @KevinRansom any thoughts?

Reply:
these are fine.   The _ will go away eventually, actually pretty soon I expect, when we no longer build the desktop version.